cove_ofds/templates/cove_ofds/jsonschema_validation_panel.html

```diff
@@ -0,0 +1,151 @@
+{% load i18n %}
+
+{% if 'prefixItems' in validation_errors %}
+    <h4>{% trans 'prefixItems' %}</h4>
+    <p>DESCIRPTION</p>
```

Comment:
Wee typo here

Reply:
It's a placeholder anyway as the source spreadsheet was blank - I'll open a issue to point @duncandewhurst to it